src/pages/profil/index.js

```diff
@@ -0,0 +1,14 @@
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+
+function Profile() {
+  const router = useRouter();
+
+  useEffect(() => {
+    //for now there is no /profil endpoint. /profil will redirect to /profil/mine-biblioteker
+    router.push("/profil/mine-biblioteker");
```

Comment:
Som Jonas har snakket om (ved ikke om han kommer til at skrive det), så skal man nok replace her, for at undgå problemer med historikken, potentielt problemer med løkker

Reply:
Har sat en replace istedet for